test/EndToEnd/tests/install.ps1

```diff
@@ -151,7 +151,7 @@ function Test-PackageWithIncompatibleAssembliesRollsInstallBack {
     $p = New-WebApplication
 
     # Act & Assert
-    Assert-Throws { Install-Package BingMapAppSDK -Project $p.Name -Source $context.RepositoryPath } "Could not install package 'BingMapAppSDK 1.0.1011.1716'. You are trying to install this package into a project that targets '.NETFramework, Version=v4.5', but the package does not contain any assembly references or content files that are compatible with that framework. For more information, contact the package author."
```

Comment:
Not sure how to add comments on lines that are hidden. Please check function InstallPackageWithUnsupportedReference at around line 428. You may have fix that test too